### PR TITLE
ranking.goo.ne.jp

### PR DESCRIPTION
**Pull request by spiri-leo**
Issue: #108335

### CODE REVIEW

File: AnnoyancesFilter/sections/push-notifications.txt [hunk 1]
```diff
@@ -444,6 +444,7 @@ abendzeitung-muenchen.de###webpush-app
 /firebase.js$domain=novayagazeta.ru|comments.ua|kinosimka.net|ilan365.net|cnbcindonesia.com|haberzamani.net|oxy.cloud|allbasketball.org|auslogics.com|kartalyuvasi.com.tr|cnnindonesia.com|fbc.ua|astv.ru
 /firebase-messaging.js$domain=focus.ua|ukranews.com|comments.ua|coub.com|porno90.me|cnnamador.com|giallozafferano.it|wcostream.com|fptplay.vn|jooble.org|mobile01.com|myinstants.com|pdalife.ru|rg.ru|m.iguides.ru|donanimhaber.com|profile.ru|womanhit.ru|news.noteru.com|narodna-pravda.ua|newstm.in|sabah.com.tr|dreamstime.com|popbee.com|avtovzglyad.ru|allbasketball.org|tw.news.yahoo.com|takvim.com.tr|jn.pt|ria.ru|pars.host|calculator-online.net|operanewsapp.com|ahaber.com.tr|tamindir.com|concursosnobrasil.com.br|kompas.com|news.ebc.net.tw|recyclingtoday.com|homedecoratione.com|khersondaily.com|mirtesen.ru|womanhit.ru|belsat.eu|gearbest.com|op.gg|novayagazeta.ru|pocket-lint.com|turunculevye.com|bloomberght.com|stardima.com|aksam.com.tr|portamur.ru
 !
+ranking.goo.ne.jp##.notification-slidedown-container
```
Comment (Yuki2718) [inline AnnoyancesFilter/sections/push-notifications.txt:447]: Remove this and instead add to the `/firebase-messaging.js` rule above, i.e. `/firebase-messaging.js$domain=focus.ua|ukranews.com|comments.ua|coub.com|porno90.me|cnnamador.com|giallozafferano.it|wcostream.com|fptplay.vn|jooble.org|mobile01.com|myinstants.com|pdalife.ru|rg.ru|m.iguides.ru|donanimhaber.com|profile.ru|womanhit.ru|news.noteru.com|narodna-pravda.ua|newstm.in|sabah.com.tr|dreamstime.com|popbee.com|avtovzglyad.ru|allbasketball.org|tw.news.yahoo.com|takvim.com.tr|jn.pt|ria.ru|pars.host|calculator-online.net|operanewsapp.com|ahaber.com.tr|tamindir.com|concursosnobrasil.com.br|kompas.com|news.ebc.net.tw|recyclingtoday.com|homedecoratione.com|khersondaily.com|mirtesen.ru|womanhit.ru|belsat.eu|gearbest.com|op.gg|novayagazeta.ru|pocket-lint.com|turunculevye.com|bloomberght.com|stardima.com|aksam.com.tr|portamur.ru|ranking.goo.ne.jp`

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/push-notifications.txt (hunk 1 above)
Comment (spiri-leo) [reply AnnoyancesFilter/sections/push-notifications.txt]: Fixed.

Comment (Yuki2718) [reply AnnoyancesFilter/sections/push-notifications.txt]: Well, I mean `ranking.goo.ne.jp##.notification-slidedown-container` is not needed. All needed is `/firebase-messaging.js$domain=ranking.goo.ne.jp`

Comment (spiri-leo) [reply AnnoyancesFilter/sections/push-notifications.txt]: Fixed.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**